modules/claims_api/app/sidekiq/claims_api/va_notify_job.rb

```diff
@@ -6,18 +6,20 @@ def perform(poa_id, rep)
       return if skip_notification_email?
 
       poa = ClaimsApi::PowerOfAttorney.find(poa_id)
+
       unless poa
         raise ::ClaimsApi::Common::Exceptions::Lighthouse::ResourceNotFound.new(
           detail: "Could not find Power of Attorney with id: #{poa_id}"
         )
       end
       if organization_filing?(poa.form_data)
         org = find_org(poa, '2122')
-        send_organization_notification(poa, org)
+        res = send_organization_notification(poa, org)
       else
         poa_code_from_form('2122a', poa)
-        send_representative_notification(poa, rep)
+        res = send_representative_notification(poa, rep)
       end
+      schedule_follow_up_check(res.id) if res.present?
```

Comment:
is there anything to consider if a res isn't present and it doesn't hit the rescue? is that possible? just want to be aware of a potential failure here?

Reply:
that would mean there was an error sending, which would get rescued in the line right after, that was my thinking at least.